### PR TITLE
Tempfix hiding account info

### PR DESCRIPTION
**Pull request by tf2manu994**
white screen with unreadable text. understandable if you dont want to merge.

Also, haven't had a chance to test this, coding on mobile is hard, might want to do that

sidenote: this actually has the send gift thing also removed, so this merges 2 things in one! in theory

i should learn how git works

[Review on Reviewable](https://reviewable.io/reviews/marshallofsound/google-play-music-desktop-player-unofficial-/310)

### CODE REVIEW

**Comment by tf2manu994** [comment]
okay lets try now

**Comment by tf2manu994** [comment]
are you serious

the problem was a stupid space before a comment
really

**Comment by MarshallOfSound** [comment]
@tf2manu994 If you use atom you can use the `linter-eslint` plugin which will tell you as you edit the code the linting mistakes you make :+1:

File: src/inject/GPMInject/interface/customUI.js [hunk 1]
```diff
@@ -34,7 +34,14 @@ window.wait(() => {
   // Settings options that won't work
   hide('[data-action="upload-music"]');
   hide('[data-action="help-and-feedback"]');
+  hide('[data-action="send-gift"]');
```
Comment (MarshallOfSound) [inline src/inject/GPMInject/interface/customUI.js]: This is also in another PR (I think yours o_0)

Be careful not to bleed commits / delta into other PR's / branches

**Comment by MarshallOfSound** [comment]
Instead of hiding the account panel entirely, we should fix the underlying issue of the user not being able to read the text.  This appears to be a result of the theming library.

**Comment by tf2manu994** [comment]
ah okay

see the send gift hiding bleeding into another pull request was a good thing hehe

i should learn git

**Pull request closed without merging**